lib/flowmailer_rails/mailer.rb

```diff
@@ -5,6 +5,7 @@ module FlowmailerRails
   class Mailer
     class NoAccessTokenError < StandardError; end
     class DeliveryError < StandardError; end
+    class TooBigMessageError < StandardError; end
```

Comment:
Makes it possible to rescue all delivery errors no matter the type :)
```suggestion
    class TooBigMessageError < DeliveryError; end
```